module/Database/src/Service/Member.php

```diff
@@ -375,6 +375,16 @@ public function getProspectiveMember(int $id): array
                         '<b>Warning:</b> Member does not study at department.',
                     ];
                 }
+
+                if (
+                    'NL20INGB0001234567' === str_replace(' ', '', strtoupper($member->getIban()))
```

Comment:
This should never be needed. The `Iban` validator already takes care of removing whitespace and uppercasing the whole string.

And checking that `$member->getIban()` is not null should be done first. Passing `null` to `strtoupper()` is deprecated.

Reply:
I'll try to repair the first then, since from my testing it looks like this is not happening

Swapped them around

Reply:
Fixed in 1f6af1609a60c259fe0ac09e2250f4c415fcf7eb